Topic-3-JSDOM/1.A.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<script>
+    const newWord = prompt("Type a word");
+    let flag;
+
+    const verifyPalindrome = (newWord) => {
+        let k = newWord.length;
+        flag = 0;
+        for (i = 0; i < newWord.length && flag != 1; i++){
+            if (newWord[i] !== newWord[k-1]){
+                flag = 1;
+            }
+            k--;
+        }
+        if (flag === 1){
```

Comment:
Test this, `if (flag) { console.log('here') } else { console.log('not here')}` and check when each block is executed.